packages/dds/cell/src/cell.ts

```diff
@@ -97,7 +97,7 @@ const snapshotFileName = "header";
  * register for these events and respond appropriately as the data is modified. `valueChanged` will be emitted
  * in response to a `set`, and `delete` will be emitted in response to a `delete`.
  */
-export class SharedCell<T extends Serializable = any> extends SharedObject<ISharedCellEvents<T>>
+export class SharedCell<T = Serializable> extends SharedObject<ISharedCellEvents<T>>
```

Comment:
i don't think this give us anything

Reply:
You're correct that 'T = Serializable' is equivalent to 'T = any'.

The one very small benefit to the `Serializable` typing is that it's preserved in the *.d.ts and docs, which communicates intent to the user (if they happen to look there.)

Unfortunately, it's not preserved in Intellisense, and so I agree the benefits are slim and may be outweighed by the risk that users repeat my `T extends Serializable` mistake.

Reply:
I think you're right that it adds more confusion than clarity.  Removing.  Thx.